src/Checker/PhpVersionChecker/CheckerStrategy/DeployYamlFilesPhpVersionStrategy.php

```diff
@@ -23,7 +23,7 @@ class DeployYamlFilesPhpVersionStrategy implements PhpVersionCheckerStrategyInte
     /**
      * @var string
      */
-    public const MESSAGE_USED_NOT_ALLOWED_PHP_VERSION = "Deploy file uses not allowed PHP image version \"%s\"\nImage tag must contain allowed PHP version (image:abc-8.0)";
+    public const MESSAGE_USED_NOT_ALLOWED_PHP_VERSION = "Deploy file uses not allowed PHP image version \"%s\"\nImage tag must contain allowed PHP version (image:abc-8.1)";
```

Comment:
We need this change to module to work properly, let's release it instead of merge.